docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -19,6 +19,14 @@ default:: undefined
 A dictionary holding the https://crossplane.io/docs/v0.13/introduction/providers.html[Crossplane providers] to be installed.
 The keys of the dict are used as names for the Provider objects.
 
+== `monitoring`
+
+[horizontal]
+type:: dictionary
+default:: `enabled: true`
```

Comment:
This way of documentation is fine for when the dictionary holds the specs of a CRD or some other data structure externally defined and documented. For cases like this, I prefer to have third level headers for each and every sub parameter (`enabled` in this case). This makes documentation easier to write and read. I can see that this is overkill here but it would lead by example for future changes.